### DISCUSSION TITLE
Where I find 'daemon.json'?

### DISCUSSION THREAD

**Discussion by carromeu** [lima-vm/lima#2270] [Q&A]
I need to edit file `daemon.json` (normally found in path `/etc/docker/daemon.json`), but do not find in Lima. Anyone can helpme?

**Comment by afbjorklund** [comment]
I think the file is normally missing, but you can create a new one (for Docker). Just mind the braces, so that it is still valid JSON..

`limactl shell docker`

`sudo vim /etc/docker/daemon.json`

`sudo systemctl restart docker`

**Comment by carromeu** [comment]
I already made this, but:

![image](https://github.com/lima-vm/lima/assets/195660/f127fc6a-a768-47df-a387-34fcc76a3b55)

Since it doesn't exist, can I create the file?

**Reply by jandubois** [reply]
~The file must be created inside the VM, not on the host.~

Sorry, I was confused. Yes, you can simply create the file.
